migrations/M240409200600CreateQueueTable.php

```diff
@@ -0,0 +1,46 @@
+<?php
+
+declare(strict_types=1);
+
+use Yiisoft\Db\Migration\MigrationBuilder;
+use Yiisoft\Db\Migration\RevertibleMigrationInterface;
+use Yiisoft\Db\Migration\TransactionalMigrationInterface;
+
+final class M240409200600CreateQueueTable implements RevertibleMigrationInterface, TransactionalMigrationInterface
+{
+    private const TABLE_PREFIX = ''; // Add table prefix if required
+    private const QUEUE_TABLE = self::TABLE_PREFIX . 'queue';
+
+    public function up(MigrationBuilder $b): void
+    {
+        $this->createAssignmentsTable($b);
```

Comment:
Suggest create table here and don't use private method.

Reply:
Should I do something here?

Reply:
sorry. I thought it was me 😕